### ISSUE TITLE
checkup-plugin-ember is incompatible with ember-template-lint 4.0 & eslint v8

### ISSUE THREAD

**Issue by runspired** [checkupjs/checkup#1158]
These projects have unfortunately moved to ESM already :)

**Comment by scalvert** [comment]
Yep, I'm aware. I have a branch that converts checkup to esm, and it's very straightforward to convert. The only thing that prevented my converting things fully at the time was the lack of support in `jest`/`ts-jest`. That said, I've been experimenting with converting to [`vitest`](https://vitest.dev/), which uses esm by default as it uses vite under the covers. This looks promising, and may provide us a nice path to convert.

cc/ @rwjblue for thoughts.
